src/main/java/hudson/plugins/tfs/TeamFoundationServerScm.java

```diff
@@ -414,6 +418,11 @@ public FilePath getModuleRoot(FilePath workspace) {
     public TeamFoundationServerRepositoryBrowser getBrowser() {
         return repositoryBrowser;
     }
+    
+    @Override
+    public @CheckForNull RepositoryBrowser<?> guessBrowser() {
+      return new TeamSystemWebAccessBrowser(serverUrl);

```

Comment:
Would you mind adjusting your IDE's indentation settings for this project to 4 spaces and amending the last commit accordingly?